### PR TITLE
Add service account domain property to service account user info.

### PR DESCRIPTION
**Pull request by danieltallentire**
Adds the service account domain to the `UserInfo` object.
Docs link to [cronofy.data.service_account.domain](https://docs.cronofy.com/developers/api/identity/userinfo/#param-cronofy.data.service_account.domain).
This is included only when `CronofyType` is `service_account`.

Made a few corrections to the class docs.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## [1.13.3]
```
Comment (danieltallentire) [inline CHANGELOG.md:1]:
```suggestion
## [1.12.3]
```

Comment (victor-cronofy) [inline CHANGELOG.md]: Is it `1.12.3`?

Comment (danieltallentire) [reply CHANGELOG.md]: thanks Victor, I also noticed it just after I'd submitted :D

**Review by Nevett — APPROVED** [review]
Tested and got valid values with a service account and no change for a regular account ✅

**Pull request merged**